http-server-netty/src/main/java/io/micronaut/http/server/netty/multipart/NettyPartData.java

```diff
@@ -91,12 +82,8 @@ public byte[] getBytes() throws IOException {
      */
     @Override
     public ByteBuffer getByteBuffer() throws IOException {
-        ByteBuf byteBuf = getByteBuf();
-        try {
-            return byteBuf.nioBuffer();
-        } finally {
-            byteBuf.release();
-        }
```

Comment:
Do we still need to release this here?  Or is the releasing handled elsewhere?

Is it possible to have a test around multi-part file uploads to check we're not leaking?

Reply:
it's released by getBytes

i think we have some multipart tests in http-server-netty (though some are also in the test suites). all tests in http-server-netty have leak detection